modules/rosa-sharing-vpc-cluster-creation.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-cluster-creation_{context}"]
+= Creating your cluster in a shared VPC
+
+The *cluster-creating AWS account* user runs the following command in their terminal to create the cluster in the shared VPC.
```

Comment:
```suggestion
To create a cluster in a shared VPC, complete the following steps. 

[NOTE]
====
Installing a cluster in a shared VPC is supported only for OpenShift 4.13.9 and later.
====

.Prerequisites

You must have the *cluster-creating AWS account* user role.

.Procedure

In a terminal, enter the following command to create a cluster in the shared VPC:

```
- This revision might be overkill, but the current format of placing the command between two admonitions (Note and Important) made it a little hard for me to figure out where to look at first. This revision adds a few more visual clues that will hopefully guide the reader well, and places the info that you'd probably want to know before you get started before the task.
- In the note text, I also moved "only" to be as close to the phrase that it modifies as possible, per IBM Style.
- One thing to check: I wasn't sure if bold highlighting was correct for the "cluster-creating AWS account" user name. I've seen user names highlighted in monospace before, but not in bold. Is that typical for ROSA docs?